### PR TITLE
Added longTap functionality

### PR DESCRIPTION
**Pull request by shibupanda**
I have built using tsc command and tested. It worked as expected.
Please review and accept the pull request. 
I attached the screen-shot where you can able to see 'increment' tooltip. (It will happen only in case of the long press)
flutter-longPress

### CODE REVIEW

**Comment by shibupanda** [comment]
Hi, @truongsinh Could you please approve this.
Thanks

File: example/nodejs/src/index.js [hunk 1]
```diff
@@ -79,6 +79,10 @@ const opts = {
   */
 
   assert.strictEqual(await driver.getElementText(counterTextFinder), '0');
+  
+  //Long Press on Increment button, it should visible 'increment' tooltip after longTap
+  await driver.execute('flutter:longTap', find.byValueKey('increment'), {durationMilliseconds: 10000, frequency: 30});
```
[the remark this answers is not in the document]
Comment (shibupanda) [reply example/nodejs/src/index.js]: Thanks, @truongsinh let me implement the same in gesture.ts. I think it's a good idea to keep both, what your opinion on this. As we can scroll using touch action and even with commands like a mobile: scroll in uiautomator2/XCUITest.

[the remark this answers is not in the document]
Comment (shibupanda) [reply example/nodejs/src/index.js]: @truongsinh Now I am implementing **longPress** in gesture.ts. Almost done need to refactor the code.
This time I will handle with or without wait action. If there is not wait action I am taking 10000 ms default. Appreciate your suggestion.

[the remark this answers is not in the document]
Comment (truongsinh) [reply example/nodejs/src/index.js]: thanks, did you push new commits?

[the remark this answers is not in the document]
Comment (shibupanda) [reply example/nodejs/src/index.js]: Doing code refactoring. It's working perfectly. 
Let me push the commits.

[the remark this answers is not in the document]
Comment (shibupanda) [reply example/nodejs/src/index.js]: @truongsinh Could you please review the latest commit. I am fixing the check errors.
Thanks

[the remark this answers is not in the document]
Comment (truongsinh) [reply example/nodejs/src/index.js]: i see that we have exposed the api, but `example/nodejs/src/index.js` is still using `await driver.execute('flutter:longTap'` right?

[the remark this answers is not in the document]
Comment (truongsinh) [reply example/nodejs/src/index.js]: ah nevermind that, the example has both

**Comment by shibupanda** [comment]
Thanks, @truongsinh. Yes, I try to make test engineers work easier by providing flutter: longTap command :smile:. Needs to update README as well.

**Pull request merged**